packages/pn-personafisica-login/src/pages/logout/Logout.tsx

```diff
@@ -1,15 +1,23 @@
 import { useEffect } from 'react';
-import { useNavigate } from 'react-router-dom';
+import {useNavigate, useSearchParams} from 'react-router-dom';
 
 import { ROUTE_LOGIN } from '../../utils/constants';
 import { storageOnSuccessOps, storageOriginOps } from '../../utils/storage';
 
 const Logout = () => {
   const navigate = useNavigate();
+  const [params] = useSearchParams();
+  const origin = params.get('origin');
+
+
 
   useEffect(() => {
     storageOnSuccessOps.delete();
     storageOriginOps.delete();
+    if (origin !== null && origin !== '') {
```

Comment:
@LorenzoCocciarelli non andrei a passare nello storage in questo punto il path dell'origin. In questo punto, nel navigate, aggiungiamo il parametro desiderato (cioè in questo caso il path). In questo componente andiamo a ripulire lo storage e non scriverci.